erpnext_com/www/docs/user/manual/en/setting-up/chart-of-accounts-importer.md

```diff
@@ -0,0 +1,42 @@
+<!--add breadcrumbs-->
+
+# Chart Of Accounts Importer
+
+>Introduced in Version 12
+
+To start importing charts of accounts navigate to
+
+> Getting Started > Data Import and Settings > Chart of Accounts Importer
+
+1\. Select the Company for which you want to import chart of accounts. Make sure the company you are selecting doesn't have an pre existing transactions otherwise you'll receive a validation error.
+
+2\. Click on `Download Template` button on the top right corner to download the template. The template you have downloaded will look as follows.
+
+<img class="screenshot" alt="POS Setting" src="{{docs_base_url}}/assets/img/setup/coa-template-download.png">
+
+<img class="screenshot" alt="POS Setting" src="{{docs_base_url}}/assets/img/setup/coa-template.png">
+
+3\. Once you download the template fill in the template as shown in the screenshot below. Please make sure to make accounts for account type "Cost of Goods Sold", "Depreciation", "Fixed Asset", "Payable", "Receivable", "Stock Adjustment". Root type for these accounts must be one of Asset, Liability, Income, Expense and Equity
```

Comment:
download the template, //add comma
fill 'details' in the template?
....to make accounts for account 'types'